INFO     [🔧TEST CONFIG]:conftest.py:87 Loading test configuration
INFO     [🔧TEST CONFIG]:conftest.py:378 Navigating to: https://console.dev.apolo.us/
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": false, "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
INFO     [🔧TEST CONFIG]:conftest.py:93 Creating data manager
INFO     [🔧TEST CONFIG]:conftest.py:123 Creating users manager
INFO     [🔧TEST CONFIG]:conftest.py:109 Creating API helper
INFO     [🔧TEST CONFIG]:conftest.py:117 Creating Apolo CLI instance
INFO     [🔧TEST CONFIG]:conftest.py:205 Continue using UserData(email='regression-ceadr7jsak@apolo.us', username='regression-ceadr7jsak', password='TM4wn6ACLjm&') as main test user...
INFO     [👤UsersManager]:users_manager.py:73 Setting main user to UserData(email='regression-ceadr7jsak@apolo.us', username='regression-ceadr7jsak', password='TM4wn6ACLjm&')
INFO     [🔧TEST CONFIG]:conftest.py:235 Continue using UserData(email='regression-nl69exwsvz@apolo.us', username='regression-nl69exwsvz', password='8ISTI6#9!ZOs') as second test user...
INFO     [👤UsersManager]:users_manager.py:82 Setting second user to UserData(email='regression-nl69exwsvz@apolo.us', username='regression-nl69exwsvz', password='8ISTI6#9!ZOs')
INFO     [🔧TEST CONFIG]:conftest.py:240 Continue using UserData(email='regression-p7oso6fqhi@apolo.us', username='regression-p7oso6fqhi', password='ICnthK!X6TGG') as third test user...
INFO     [👤UsersManager]:users_manager.py:91 Setting third user to UserData(email='regression-p7oso6fqhi@apolo.us', username='regression-p7oso6fqhi', password='ICnthK!X6TGG')
INFO     [📘TEST_INFO]:reporting.py:86 ------------------------------------------------------------
INFO     [📘TEST_INFO]:reporting.py:87 📝 TEST started: Invite not registered user as user to organization
INFO     [📘TEST_INFO]:reporting.py:88 ------------------------------------------------------------
INFO     [🔧TEST CONFIG]:conftest.py:378 Navigating to: https://console.dev.apolo.us/
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": false, "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Login via UI
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click login button
INFO     [🎭 Playwright]:base_page.py:16 <AuthPage>: Click log in button
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click login button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_login_button_success_3cd3cb.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter email
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter regression-ceadr7jsak@apolo.us email
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter email
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_email_success_047f66.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter password
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter ********* password
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter password
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_password_success_1155a4.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click Continue button
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Click continue button
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": true, "orgs": [], "clusters": [], "projects": [], "admin_url": "https://api.dev.apolo.us/apis/admin/v1", "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Waiting for spinner
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Waiting for spinner to disappear
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/users/regression-ceadr7jsak/permissions?uri=blob:
Request Body:
<not available>
Response Body (first 1000 chars):
[]
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/apis/admin/v1/users/regression-ceadr7jsak?include=clusters&include=system_role
Request Body:
<not available>
Response Body (first 1000 chars):
{"name": "regression-ceadr7jsak", "email": "regression-ceadr7jsak@apolo.us", "first_name": null, "last_name": null, "created_at": "2025-09-16T01:32:44.055799+00:00", "terms_accepted": true, "clusters": []}
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 500] GET https://api.dev.apolo.us/apis/admin/v1/invites/incoming
Request Body:
<not available>
Response Body (first 1000 chars):
{"error": "Unexpected exception: (sqlalchemy.dialects.postgresql.asyncpg.ProgrammingError) <class 'asyncpg.exceptions.UndefinedColumnError'>: column users.email does not exist\n[SQL: SELECT users.name, users.metadata, users.email, users.version \nFROM users \nWHERE users.name = %s]\n[parameters: ('regression-ceadr7jsak',)]\n(Background on this error at: https://sqlalche.me/e/14/f405). Path with query: /apis/admin/v1/invites/incoming."}
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 500] GET https://api.dev.apolo.us/apis/admin/v1/users/regression-ceadr7jsak?include=orgs
Request Body:
<not available>
Response Body (first 1000 chars):
{"error": "Unexpected exception: (sqlalchemy.dialects.postgresql.asyncpg.ProgrammingError) <class 'asyncpg.exceptions.UndefinedColumnError'>: column users.email does not exist\n[SQL: SELECT users.name, users.metadata, users.email, users.version \nFROM users \nWHERE users.name = %s]\n[parameters: ('regression-ceadr7jsak',)]\n(Background on this error at: https://sqlalche.me/e/14/f405). Path with query: /apis/admin/v1/users/regression-ceadr7jsak?include=orgs."}
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click Continue button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_continue_button_success_044d7a.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify that Welcome new user page displayed
INFO     [🎭 Playwright]:base_page.py:16 <WelcomeNewUserPage>: Check if page loaded
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Verify that Welcome new user page displayed
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/verify_ui_page_displayed_success_e21284.png
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Login via UI
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_login_success_9142d4.png
INFO     [👤UsersManager]:users_manager.py:59 Generated new user: UserData(email='regression-6v0oyo6d5y@apolo.us', username='regression-6v0oyo6d5y', password='4svIiQZj#ccL')
INFO     [👤UsersManager]:users_manager.py:73 Setting main user to UserData(email='regression-6v0oyo6d5y@apolo.us', username='regression-6v0oyo6d5y', password='4svIiQZj#ccL')
INFO     [📄 TestCaseInfo]:base_test_class.py:83 ::::::::::::::: User1 pass onboarding and create organization
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Add organization via API and reload page
INFO     [🔧DATA MANAGER]:test_data.py:56 Added organization: org_name=regression-org-ob3w0ithddtw, gherkin_name=Default-organization
INFO     [🌐API_helper]:api_helper.py:86 POST https://api.dev.apolo.us/apis/admin/v1/orgs?skip_auto_add_to_clusters=false with data: {'name': 'regression-org-ob3w0ithddtw'}
INFO     [🌐API_helper]:api_helper.py:190 Add org regression-org-ob3w0ithddtw response: {'name': 'regression-org-ob3w0ithddtw', 'balance': {'credits': '500', 'spent_credits': '0'}, 'user_default_credits': None, 'notification_intervals': {'balance_projection_seconds': [604800, 259200, 86400], 'balance_amount': [0], 'balance_depletion_seconds': [86400, 604800]}}
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Reload page
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": true, "orgs": [{"name": "regression-org-ob3w0ithddtw", "role": "admin"}], "clusters": [{"name": "default", "registry_url": "https://registry.api.dev.apolo.us", "storage_url": "https://api.dev.apolo.us/api/v1/storage", "monitoring_url": "https://monitoring.api.dev.apolo.us/api/v1/jobs", "secrets_url": "https://api.dev.apolo.us/api/v1/secrets", "metrics_url": "https://grafana.dev.apolo.us", "disks_url": "https://api.dev.apolo.us/api/v1/disk", "buckets_url": "https://api.dev.apolo.us/api/v1/buckets", "resource_pool_types": [{"name": "a2-highgpu-1g", "min_size": 0, "max_size": 10, "cpu": 12.0, "available_cpu": 11.081, "memory": 89629032448, "available_memory": 80898240256, "disk_size": 523982073856, "available_disk_size": 364209683290, "nvidia_gpu": {"count": 1, "model": "NVIDIA-A100-SXM4-40GB", "memory": 42949672960}}, {"name": "a2-highgpu-2g", "min_size": 0, "max_size": 3, "cpu": 24, "available_cpu": 23.3, "memory": 166599852032, "available_memory": 154401264108, "disk_siz
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/users/regression-ceadr7jsak/permissions?uri=blob:
Request Body:
<not available>
Response Body (first 1000 chars):
[{"uri": "blob://default/regression-org-ob3w0ithddtw", "action": "manage"}]
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/apis/admin/v1/orgs/regression-org-ob3w0ithddtw
Request Body:
<not available>
Response Body (first 1000 chars):
{"name": "regression-org-ob3w0ithddtw", "balance": {"credits": "500", "spent_credits": "0"}, "user_default_credits": null, "notification_intervals": {"balance_projection_seconds": [604800, 259200, 86400], "balance_amount": [0], "balance_depletion_seconds": [86400, 604800]}}
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 500] GET https://api.dev.apolo.us/apis/admin/v1/invites/incoming
Request Body:
<not available>
Response Body (first 1000 chars):
{"error": "Unexpected exception: (sqlalchemy.dialects.postgresql.asyncpg.ProgrammingError) <class 'asyncpg.exceptions.UndefinedColumnError'>: column users.email does not exist\n[SQL: SELECT users.name, users.metadata, users.email, users.version \nFROM users \nWHERE users.name = %s]\n[parameters: ('regression-ceadr7jsak',)]\n(Background on this error at: https://sqlalche.me/e/14/f405). Path with query: /apis/admin/v1/invites/incoming."}
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/apis/admin/v1/users/regression-ceadr7jsak?include=clusters&include=system_role
Request Body:
<not available>
Response Body (first 1000 chars):
{"name": "regression-ceadr7jsak", "email": "regression-ceadr7jsak@apolo.us", "first_name": null, "last_name": null, "created_at": "2025-09-16T01:32:44.055799+00:00", "terms_accepted": true, "clusters": [{"cluster_name": "default", "role": "admin", "org_name": "regression-org-ob3w0ithddtw", "balance": {"spent_credits": "0"}}]}
INFO     [🎭 Playwright]:base_page.py:16 <MainPage>: Waiting for spinner
INFO     [🎭 Playwright]:base_page.py:16 <MainPage>: Waiting for spinner to disappear
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 500] GET https://api.dev.apolo.us/apis/admin/v1/users/regression-ceadr7jsak?include=orgs
Request Body:
<not available>
Response Body (first 1000 chars):
{"error": "Unexpected exception: (sqlalchemy.dialects.postgresql.asyncpg.ProgrammingError) <class 'asyncpg.exceptions.UndefinedColumnError'>: column users.email does not exist\n[SQL: SELECT users.name, users.metadata, users.email, users.version \nFROM users \nWHERE users.name = %s]\n[parameters: ('regression-ceadr7jsak',)]\n(Background on this error at: https://sqlalche.me/e/14/f405). Path with query: /apis/admin/v1/users/regression-ceadr7jsak?include=orgs."}
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Reload page
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_reload_page_success_447e54.png
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Add organization via API and reload page
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_add_org_api_success_3375eb.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Invite user to organization via UI
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click User Organization settings button
INFO     [🎭 Playwright]:base_page.py:16 <MainPage>: Click organization settings button
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click User Organization settings button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_organization_settings_button_success_9bc437.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify that Organization settings pop up displayed
INFO     [🎭 Playwright]:base_page.py:16 <OrganizationSettingsPopup>: Check if page loaded
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Verify that Organization settings pop up displayed
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/verify_ui_popup_displayed_success_793cab.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click People button
INFO     [🎭 Playwright]:base_page.py:16 <OrganizationSettingsPopup>: Click People button
INFO     [🎭 Playwright]:base_page.py:16 <OrganizationSettingsPopup>: Waiting for spinner
INFO     [🎭 Playwright]:base_page.py:16 <OrganizationSettingsPopup>: Waiting for spinner to disappear
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 404] GET https://api.dev.apolo.us/apis/admin/v1/orgs/regression-org-ob3w0ithddtw/users?with_user_info=true
Request Body:
<not available>
Response Body (first 1000 chars):
{"error": "Org 'regression-org-ob3w0ithddtw' not found"}
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 404] GET https://api.dev.apolo.us/apis/admin/v1/orgs/regression-org-ob3w0ithddtw/users/regression-ceadr7jsak
Request Body:
<not available>
Response Body (first 1000 chars):
{"error": "User 'regression-ceadr7jsak' not found in org 'regression-org-ob3w0ithddtw'"}
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/apis/admin/v1/invites
Request Body:
<not available>
Response Body (first 1000 chars):
[]
INFO     [🎭 Playwright]:base_page.py:16 <OrganizationSettingsPopup>: Timeout error for spinner
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click People button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_people_button_success_22b5b4.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify that Organization people page displayed
INFO     [🎭 Playwright]:base_page.py:16 <OrganizationPeoplePage>: Check if page loaded
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Verify that Organization people page displayed
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/verify_ui_page_displayed_success_7de6ab.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click Invite people button
INFO     [🎭 Playwright]:base_page.py:16 <OrganizationPeoplePage>: Click Invite people button
ERROR    [📘TEST_INFO]:reporting.py:56 ❌ STEP failed: Click Invite people button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (fail): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_invite_people_button_fail_5e08f2.png
ERROR    [📘TEST_INFO]:reporting.py:56 ❌ STEP failed: Invite user to organization via UI
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (fail): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_invite_user_to_org_fail_7d8843.png
ERROR    [📘TEST_INFO]:reporting.py:95 ❌ TEST failed: Invite not registered user as user to organization
Exception: [AssertionError] during step: Invite user to organization via UI
Message: [TimeoutError] during step: Click Invite people button
Message: Element not ready for click after 10000ms: <Locator frame=<Frame name= url='https://console.dev.apolo.us/org-settings?org=regression-org-ob3w0ithddtw'> selector='button >> internal:has-text="Invite people"i'>
Last error: Locator expected to be enabled
Actual value: disabled 
Call log:
  - Expect "to_be_enabled" with timeout 500ms
  - waiting for locator("button").filter(has_text="Invite people")
    4 × locator resolved to <button disabled type="button" data-state="closed" aria-expanded="false" aria-haspopup="dialog" aria-controls="radix-:rn:" class="relative inline-flex cursor-pointer items-center justify-center rounded-xl py-1 px-5 transition-colors text-button h-12 bg-rebecca text-white hover:bg-rebecca-dark active:bg-rebecca disabled:bg-neural-01 disabled:text-neural-03 pl-4 gap-3 capitalize pointer-events-none">…</button>
      - unexpected value "disabled"

INFO     [📘TEST_INFO]:reporting.py:98 ------------------------------------------------------------
INFO     [🌐API_helper]:api_helper.py:54 GET https://api.dev.apolo.us/apis/admin/v1/orgs
INFO     [🌐API_helper]:api_helper.py:165 Status: 200. Response: []
INFO     [🔧TEST CONFIG]:conftest.py:291 Cleaning up 0 organisations
INFO     [🔧TEST CONFIG]:conftest.py:321 Closing all browser sessions
INFO     [🔧TEST CONFIG]:conftest.py:327 Stopped Playwright
WARNING  [🔧TEST CONFIG]:conftest.py:337 Failed to close context: BrowserContext.close: Target page, context or browser has been closed
INFO     [🔧TEST CONFIG]:conftest.py:342 Closed browser (post-stop)
INFO     [🔧TEST CONFIG]:conftest.py:327 Stopped Playwright
WARNING  [🔧TEST CONFIG]:conftest.py:337 Failed to close context: BrowserContext.close: Target page, context or browser has been closed
INFO     [🔧TEST CONFIG]:conftest.py:342 Closed browser (post-stop)
INFO     [🔧TEST CONFIG]:conftest.py:347 Browser cleanup finished
INFO     [🔧TEST CONFIG]:conftest.py:87 Loading test configuration
INFO     [🔧TEST CONFIG]:conftest.py:378 Navigating to: https://console.dev.apolo.us/
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": false, "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
INFO     [🔧TEST CONFIG]:conftest.py:93 Creating data manager
INFO     [🔧TEST CONFIG]:conftest.py:123 Creating users manager
INFO     [🔧TEST CONFIG]:conftest.py:109 Creating API helper
INFO     [🔧TEST CONFIG]:conftest.py:117 Creating Apolo CLI instance
INFO     [🔧TEST CONFIG]:conftest.py:205 Continue using UserData(email='regression-ceadr7jsak@apolo.us', username='regression-ceadr7jsak', password='TM4wn6ACLjm&') as main test user...
INFO     [👤UsersManager]:users_manager.py:73 Setting main user to UserData(email='regression-ceadr7jsak@apolo.us', username='regression-ceadr7jsak', password='TM4wn6ACLjm&')
INFO     [🔧TEST CONFIG]:conftest.py:235 Continue using UserData(email='regression-nl69exwsvz@apolo.us', username='regression-nl69exwsvz', password='8ISTI6#9!ZOs') as second test user...
INFO     [👤UsersManager]:users_manager.py:82 Setting second user to UserData(email='regression-nl69exwsvz@apolo.us', username='regression-nl69exwsvz', password='8ISTI6#9!ZOs')
INFO     [🔧TEST CONFIG]:conftest.py:240 Continue using UserData(email='regression-p7oso6fqhi@apolo.us', username='regression-p7oso6fqhi', password='ICnthK!X6TGG') as third test user...
INFO     [👤UsersManager]:users_manager.py:91 Setting third user to UserData(email='regression-p7oso6fqhi@apolo.us', username='regression-p7oso6fqhi', password='ICnthK!X6TGG')
INFO     [📘TEST_INFO]:reporting.py:86 ------------------------------------------------------------
INFO     [📘TEST_INFO]:reporting.py:87 📝 TEST started: Invite not registered user as user to organization
INFO     [📘TEST_INFO]:reporting.py:88 ------------------------------------------------------------
INFO     [🔧TEST CONFIG]:conftest.py:378 Navigating to: https://console.dev.apolo.us/
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": false, "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Login via UI
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click login button
INFO     [🎭 Playwright]:base_page.py:16 <AuthPage>: Click log in button
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click login button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_login_button_success_05b17d.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter email
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter regression-ceadr7jsak@apolo.us email
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter email
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_email_success_13d687.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter password
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter ********* password
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter password
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_password_success_00ee77.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click Continue button
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Click continue button
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Waiting for spinner
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": true, "orgs": [{"name": "regression-org-ob3w0ithddtw", "role": "admin"}], "clusters": [{"name": "default", "registry_url": "https://registry.api.dev.apolo.us", "storage_url": "https://api.dev.apolo.us/api/v1/storage", "monitoring_url": "https://monitoring.api.dev.apolo.us/api/v1/jobs", "secrets_url": "https://api.dev.apolo.us/api/v1/secrets", "metrics_url": "https://grafana.dev.apolo.us", "disks_url": "https://api.dev.apolo.us/api/v1/disk", "buckets_url": "https://api.dev.apolo.us/api/v1/buckets", "resource_pool_types": [{"name": "a2-highgpu-1g", "min_size": 0, "max_size": 10, "cpu": 12.0, "available_cpu": 11.081, "memory": 89629032448, "available_memory": 80898240256, "disk_size": 523982073856, "available_disk_size": 364209683290, "nvidia_gpu": {"count": 1, "model": "NVIDIA-A100-SXM4-40GB", "memory": 42949672960}}, {"name": "a2-highgpu-2g", "min_size": 0, "max_size": 3, "cpu": 24, "available_cpu": 23.3, "memory": 166599852032, "available_memory": 154401264108, "disk_siz
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/users/regression-ceadr7jsak/permissions?uri=blob:
Request Body:
<not available>
Response Body (first 1000 chars):
[{"uri": "blob://default/regression-org-ob3w0ithddtw", "action": "manage"}]
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 404] GET https://api.dev.apolo.us/apis/admin/v1/orgs/regression-org-ob3w0ithddtw
Request Body:
<not available>
Response Body (first 1000 chars):
{"error": "Org 'regression-org-ob3w0ithddtw' not found"}
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Waiting for spinner to disappear
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 500] GET https://api.dev.apolo.us/apis/admin/v1/invites/incoming
Request Body:
<not available>
Response Body (first 1000 chars):
{"error": "Unexpected exception: (sqlalchemy.dialects.postgresql.asyncpg.ProgrammingError) <class 'asyncpg.exceptions.UndefinedColumnError'>: column users.email does not exist\n[SQL: SELECT users.name, users.metadata, users.email, users.version \nFROM users \nWHERE users.name = %s]\n[parameters: ('regression-ceadr7jsak',)]\n(Background on this error at: https://sqlalche.me/e/14/f405). Path with query: /apis/admin/v1/invites/incoming."}
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/apis/admin/v1/users/regression-ceadr7jsak?include=orgs
Request Body:
<not available>
Response Body (first 1000 chars):
{"name": "regression-ceadr7jsak", "email": "regression-ceadr7jsak@apolo.us", "first_name": null, "last_name": null, "created_at": "2025-09-16T01:32:44.055799+00:00", "terms_accepted": true, "orgs": [{"org_name": "regression-org-ob3w0ithddtw", "role": "admin", "balance": {"spent_credits": "0"}}]}
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/apis/admin/v1/users/regression-ceadr7jsak?include=clusters&include=system_role
Request Body:
<not available>
Response Body (first 1000 chars):
{"name": "regression-ceadr7jsak", "email": "regression-ceadr7jsak@apolo.us", "first_name": null, "last_name": null, "created_at": "2025-09-16T01:32:44.055799+00:00", "terms_accepted": true, "clusters": [{"cluster_name": "default", "role": "admin", "org_name": "regression-org-ob3w0ithddtw", "balance": {"spent_credits": "0"}}]}
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Timeout error for spinner
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click Continue button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_continue_button_success_b7fcc3.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify that Welcome new user page displayed
INFO     [🎭 Playwright]:base_page.py:16 <WelcomeNewUserPage>: Check if page loaded
ERROR    [📘TEST_INFO]:reporting.py:56 ❌ STEP failed: Verify that Welcome new user page displayed
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (fail): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/verify_ui_page_displayed_fail_7f3603.png
ERROR    [📘TEST_INFO]:reporting.py:56 ❌ STEP failed: Login via UI
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (fail): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_login_fail_3b1d0c.png
ERROR    [📘TEST_INFO]:reporting.py:95 ❌ TEST failed: Invite not registered user as user to organization
Exception: [AssertionError] during step: Login via UI
Message: [AssertionError] during step: Verify that Welcome new user page displayed
Message: Locator expected to be visible
Actual value: <element(s) not found> 
Call log:
  - Expect "to_be_visible" with timeout 5000ms
  - waiting for locator("h3.truncate.text-h3").filter(has_text="Welcome, regression-ceadr7jsak@apolo.us")

INFO     [📘TEST_INFO]:reporting.py:98 ------------------------------------------------------------
INFO     [🌐API_helper]:api_helper.py:54 GET https://api.dev.apolo.us/apis/admin/v1/orgs
INFO     [🌐API_helper]:api_helper.py:165 Status: 200. Response: []
INFO     [🔧TEST CONFIG]:conftest.py:291 Cleaning up 0 organisations
INFO     [🔧TEST CONFIG]:conftest.py:321 Closing all browser sessions
INFO     [🔧TEST CONFIG]:conftest.py:327 Stopped Playwright
WARNING  [🔧TEST CONFIG]:conftest.py:337 Failed to close context: BrowserContext.close: Target page, context or browser has been closed
INFO     [🔧TEST CONFIG]:conftest.py:342 Closed browser (post-stop)
INFO     [🔧TEST CONFIG]:conftest.py:327 Stopped Playwright
WARNING  [🔧TEST CONFIG]:conftest.py:337 Failed to close context: BrowserContext.close: Target page, context or browser has been closed
INFO     [🔧TEST CONFIG]:conftest.py:342 Closed browser (post-stop)
INFO     [🔧TEST CONFIG]:conftest.py:347 Browser cleanup finished